src/features/Chat/components/ActiveWindow/Header/index.tsx

```diff
@@ -86,11 +86,12 @@ const Header = ({ chat }: Props) => {
       title: t(`dialog.${variant}.title`),
     });
     if (isConfirmed) {
-      updateChatStatus({
+      await updateChatStatus({
         userId,
         buddyId: chat.buddyId,
         status: confirmDialogMap[variant].targetFolder,
-      });
+      }).unwrap();
+      if (variant === 'block') dispatch(clearActiveChat());
```

Comment:
What happens if the variant is "delete"?

Reply:
The only variants that function accepts are "archive", "block", "restore" and "unblock".

Reply:
alright great